internal/packages/packages.go

```diff
@@ -19,6 +24,15 @@ type PackageManifest struct {
 	Version string `json:"version"`
 }
 
+// DatasetManifest represents the structure of a dataset's manifest
+type DatasetManifest struct {
```

Comment:
Yes, this might happen once the spec starts to implement file contents' validation, the remaining item in https://github.com/elastic/package-spec/issues/4.